config/config-windows.yaml

```diff
@@ -26,7 +26,41 @@ receivers:
       swap:
       process:
 
-  # agent self-observability metrics
+  # scrape iis perf counters at 1m interval
+  windowsperfcounters/iis:
+    collection_interval: 1m
+    perfcounters:
+      - object: Web Service
+        instances: _Total
+        counters:
+          # TODO: instead of setting capture group values to lowercase, make this option explicit
+          - Current Connections
```

Comment:
I suppose these names are defined somewhere as constants?
Might be useful to add a link in a comment.

Reply:
There counter names are defined by Microsoft. It's actually really hard to find a good link, but some relevant docs will be added here: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/receiver/windowsperfcountersreceiver#iis